src/blog/2019/one-body-many-members.md

```diff
@@ -0,0 +1,33 @@
+---
+title: One Body, Many Members
+tags: Build
+author: Andy Geers
+date: 2019-10-01
+intro:
+---
+
+Last month at Kingdom Code BUILD we spent some time dwelling on the following words from Romans 12:3-9 (NIV):
+
+> "For by the grace given me I say to every one of you: Do not think of yourself more highly than you ought, but rather think of yourself with sober judgment, in accordance with the faith God has distributed to each of you. For just as each of us has one body with many members, and these members do not all have the same function, o in Christ we, though many, form one body, and each member belongs to all the others. We have different gifts, according to the grace given to each of us. If your gift is prophesying, then prophesy in accordance with your faith; if it is serving, then serve; if it is teaching, then teach; if it is to encourage, then give encouragement; if it is giving, then give generously; if it is to lead, do it diligently; if it is to show mercy, do it cheerfully. Love must be sincere. Hate what is evil; cling to what is good."
+
+
+## 1. Within Kingdom Code we are one body with many members
+
+One of the glorious things about Kingdom Code is the range of different skills represented within the community. Something we often hear when people find out about Kingdom Code is “Oh, but I’m not a coder…” - but praise God that we’re not all coders! A good hackathon team benefits massively from a range of input, from designers to project managers… and yes, ultimately they usually need coders too!
```

Comment:
Can you pull this out a little bit wider than a hackathon context for this blog post? Perhaps something along the lines of 'a good product team'?